code/modules/recycling/conveyor2.dm

```diff
@@ -41,6 +41,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	. = ..()
 	operating = TRUE
 	update_move_direction()
+	begin_processing()
```

Comment:
change the subtype's processing_flags to START_PROCESSING_ON_INIT instead

also, update icon

Reply:
All set